src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -794,18 +794,18 @@ export const LedgerCryptoProvider: (transport: Transport) => Promise<CryptoProvi
   })
 
   const signVotingRegistrationMetaData = async (
```

Comment:
Shouldn't this be `signCVoteRegistrationMetaData`? Also, there is a lot of other `VotingRegistration` occurences still present in the code.